src/main/js/components/LogIn.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="login">
+    <input name="email" type="email" placeholder="Email Address">
+    <input name="password" type="password" placeholder="Password">
+    {{ incorrect_user_info }}
+    <input type="button" v-on:click="login" value="Log In">
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+
+export default {
+  data() {
+    return {
+      user: { email: null, password: null },
+      incorrect_user_info: null
+    }
+  },
+  methods: {
```

Comment:
ここにcsrfトークンを取得するメソッドも書いて欲しい
```suggestion
  methods: {
    csrf: function(){
      const csrf = axios.get("/prelogin");
      return csrf;
  }
```
だったっけな